src/app/components/layouts/expandable-text/expandable-text.component.html

```diff
@@ -0,0 +1,14 @@
+<div id="{{id}}" *ngIf="!expanded; else expandedTemplate" [clamp]="lines">
+    <ng-content *ngTemplateOutlet="expandedTemplate">
```

Comment:
ng-container wäre hier glaube ich besser! So wird es auch in der Angular Doku (https://angular.io/api/common/NgTemplateOutlet) verwendet